vm.tf

```diff
@@ -210,6 +215,8 @@ export USERMGTIMAGE=vision-usermgt:${var.vision_version}
 export COS_BUCKET_BASE=${var.cos_bucket_base}
 export URLPAIVIMAGES="$${COS_BUCKET_BASE}/${var.vision_tar_name}"
 export URLPAIVDEB="$${COS_BUCKET_BASE}/${var.vision_deb_name}"
+export URLPAIVDATASET="$${COS_BUCKET_BASE}/${var.example_dataset_name}"
+export DATASET_NAME="${var.example_dataset_name}"
```

Comment:
This takes a couple of args, and also uses two different environment variables (`URLPAIVDATASET` and `DATASET_NAME`). If you choose to make the change to make the sample dataset a full URL, then this DATASET_NAME variable will be different. You don't really need the name here; why not simply save the file explicitly as "dataset.zip" with the appropriate flag for wget or curl, and then hard-code that as a known constant in the download bash script? That means you won't need this in the constants.